vnext/apps/blog/src/styles/blog.css

```
@@ -1160,24 +1160,40 @@ body.noema-public :focus-visible {
 }
 
 .article-page {
+  --assistant-panel-width: 28rem;
+  display: grid;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)